core/modules/ccontrol/MergingHandler.cc

```diff
@@ -198,6 +198,11 @@ bool MergingHandler::reset() {
     return true;
 }
 
+
+bool MergingHandler::scrubResults(int jobId, int attempt) {
+    return _infileMerger->scrubResults(jobId, attempt);
```

Comment:
I don't think there's any way I can know that at this point. Later on, it checks if the table has been created before trying to drop any rows. It also adds it to the set of jobAttemps that should not be added to the result table. The idea being, if job 20 attempt 0 had to be restarted as job 20 attempt 1, no rows from job 20 attempt 0 should be accepted if a worker comes back from a network outage.